miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/signatureverifier/api/SignatureApiClient.kt

```diff
@@ -8,7 +8,7 @@ import okhttp3.Request
 import okhttp3.Response
 import java.lang.IllegalArgumentException
 
-internal class ApiClient(baseUrl: String, subscriptionKey: String, context: Context) {
+internal class SignatureApiClient(baseUrl: String, subscriptionKey: String, context: Context) {
```

Comment:
As the `baseUrl` and `subscriptionKey` is same as the `ApiClient` we already used in miniapp sdk, we can use that ApiClient to download and verify the `miniapp`, We can create `SignatureVerificationFetcher` class to call the verify method from miniapp,then we dont have to use two `ApiClient` for the same sdk..that's a suggestion, if it make things complicated you can ignore it...🙂

Reply:
Yeah, we can align with `ApiClient`. I will refactor it in another PR.